synapse/rest/media/v1/preview_url_resource.py

```diff
@@ -761,3 +762,16 @@ def _is_html(content_type: str) -> bool:
 
 def _is_json(content_type: str) -> bool:
     return content_type.lower().startswith("application/json")
+
+
+def _is_previewable(content_type: bytes) -> bool:
+    """Returns True for content types for which we will perform URL preview and False
+    otherwise."""
+
+    content_type = content_type.lower()
+    return (
+        content_type.startswith(b"text/html")
+        or content_type.startswith(b"application/xhtml")
+        or content_type.startswith(b"image/")
+        or content_type.startswith(b"application/json")
+    )
```

Comment:
Probably best to refer to our helpers that already exist so we can update just a single spot for each of them in the future.

```suggestion
    return (
        _is_media(content_type)
        or _is_html(content_type)
        or _is_json(content_type)
    )
```